src/Command/ZaakCommand.php

```diff
@@ -99,6 +111,14 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return Command::FAILURE;
         }
 
+        $this->session->remove('currentActionUserId');
```

Comment:
waarom doen we dit hiero?
Wil je dit altijd doen, zelfs als we geen nieuwe user setten?

Reply:
Standaard vanuit core meegenomen